packages/code-du-travail-frontend/src/outils/Components/SimulatorDecorator/index.tsx

```diff
@@ -0,0 +1,114 @@
+import { theme, Wrapper } from "@socialgouv/cdtn-ui";
+import React from "react";
+import { Form } from "react-final-form";
+import styled from "styled-components";
+
+import {
+  DebugInfo,
+  FormStateToReducer,
+  Navigation,
+  StepList,
+  Title,
+} from "./Components";
+import type {
+  NavigationProps,
+  StepListProps,
+  TitleProps,
+} from "./Components/types";
+import { Decorator, FormApi, Mutator, ValidationErrors } from "final-form";
+
+type Props<FormValues> = {
+  title: TitleProps;
+  navigation: Omit<Omit<NavigationProps, "hasError">, "onPrevious"> & {
+    onPrevious?: (values: FormValues) => void;
+  };
+  steps: Omit<StepListProps, "width">;
+  initialValues?: FormValues;
+  onFormStateUpdate?: (values: FormValues) => void;
+  onFormStepSubmit: (values: FormValues, form: FormApi<FormValues>) => void;
+  showDebug: boolean;
+  validate?: (
+    values: FormValues
+  ) => ValidationErrors | Promise<ValidationErrors>;
+  decorators?: Array<Decorator<FormValues, Partial<FormValues>>>;
+  mutators?: { [key: string]: Mutator<FormValues, Partial<FormValues>> };
+  Annotations?: () => JSX.Element;
```

Comment:
```suggestion
  annotations?: JSX.Element;
```

J'utiliserai plus un composant directement qu'une fonction. De plus j'aurai laissé en miniscule, mais à voir si des règles existent sur ça

Reply:
modifié :)